packages/angular/src/generators/application/schema.d.ts

```diff
@@ -26,4 +26,5 @@ export interface Schema {
   skipPackageJson?: boolean;
   skipDefaultProject?: boolean;
   standalone?: boolean;
+  addNXWelcomeComponent?: boolean;
```

Comment:
Wouldn't it be (more) logical, to call the option something like: `skipNxWelcomeComponent`? With a default of `false`.
That way it would be more in line with the other `skip...` options.

Reply:
Yeah, you are right. That does make it more logical and uniform to what exists already.
I'll make those changes.

Reply:
Updated.